### PR TITLE
MNT: Import qt from enaml first.

### PR DESCRIPTION
**Pull request by ericdill**
pandas was importing matplotlib which was importing qt which was
causing spi API issues.

note that the `__all__` is negotiable

### CODE REVIEW

**Comment by ericdill** [comment]
I was getting this error without the code in this PR

```
(nikea2)edill@edill-810g:~/dev/python/dataportal (master *$ u=)$ replay --small-screen
Traceback (most recent call last):
  File "/home/edill/anaconda/envs/nikea2/bin/replay", line 9, in <module>
    load_entry_point('dataportal==0.0.5.post0', 'console_scripts', 'replay')()
  File "/home/edill/anaconda/envs/nikea2/lib/python2.7/site-packages/setuptools-14.3.1-py2.7.egg/pkg_resources/__init__.py", line 522, in load_entry_point

  File "/home/edill/anaconda/envs/nikea2/lib/python2.7/site-packages/setuptools-14.3.1-py2.7.egg/pkg_resources/__init__.py", line 2647, in load_entry_point

  File "/home/edill/anaconda/envs/nikea2/lib/python2.7/site-packages/setuptools-14.3.1-py2.7.egg/pkg_resources/__init__.py", line 2320, in load

  File "/home/edill/anaconda/envs/nikea2/lib/python2.7/site-packages/setuptools-14.3.1-py2.7.egg/pkg_resources/__init__.py", line 2326, in resolve

  File "/home/edill/dev/python/dataportal/dataportal/replay/replay.py", line 2, in <module>
    from enaml.qt.qt_application import QtApplication
  File "/home/edill/dev/python/enaml/enaml/qt/__init__.py", line 28, in <module>
    prepare_pyqt()
  File "/home/edill/dev/python/enaml/enaml/qt/__init__.py", line 13, in prepare_pyqt
    sip.setapi('QDate', 2)
ValueError: API 'QDate' has already been set to version 1
```

File: dataportal/__init__.py [hunk 1]
```diff
@@ -5,7 +5,12 @@
 
 logger = logging.getLogger(__name__)
 __version__ = 'v0.0.5.post0'
+# import qt from enaml to make sure that the Qt API version gets set correctly
+from enaml import qt
```
Comment (tacaswell) [inline dataportal/__init__.py]: please wrap this in a

``` python
try:
    from enaml import qt
except ImportError:
    continue
else:
   del qt
```

File: dataportal/__init__.py [hunk 2]
```diff
@@ -6,6 +6,17 @@
 logger = logging.getLogger(__name__)
 __version__ = 'v0.0.5.post0'
 
+# import qt from enaml to make sure that the Qt API version gets set correctly
+try:
+    from enaml import qt
+except ImportError:
+    pass
+else:
+   del qt
```
Comment (danielballan) [inline dataportal/__init__.py]: It looks like this line has the wrong indentation.

**Comment by klauer** [comment]
Seeing something similar on the HXN with version 0.0.6:

``` python
In [1]: import dataportal
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-8-55e2331cc9d7> in <module>()
----> 1 import dataportal

/nfs/xf03id/mc/envs/ophyd/lib/python2.7/site-packages/dataportal/__init__.py in <module>()
      9 # import qt from enaml to make sure that the Qt API version gets set correctly
     10 try:
---> 11     from enaml import qt
     12 except ImportError:
     13     pass

/nfs/xf03id/mc/envs/ophyd/lib/python2.7/site-packages/enaml/qt/__init__.py in <module>()
     26     try:
     27         import PyQt4
---> 28         prepare_pyqt()
     29         QT_API = os.environ['QT_API'] = 'pyqt'
     30     except ImportError:

/nfs/xf03id/mc/envs/ophyd/lib/python2.7/site-packages/enaml/qt/__init__.py in prepare_pyqt()
     11 def prepare_pyqt():
     12     import sip
---> 13     sip.setapi('QDate', 2)
     14     sip.setapi('QDateTime', 2)
     15     sip.setapi('QString', 2)

ValueError: API 'QDate' has already been set to version 1
```

Reference [issue](https://github.com/enthought/traits-enaml/issues/5) from enaml.

**Comment by danielballan** [comment]
Fixed in a PR

**Comment by danielballan** [comment]
Thanks for the report, Ken.

**Comment by tacaswell** [comment]
@danielballan Link to PR this was fixed in?

**Comment by danielballan** [comment]
#157

**Pull request merged**